src/rpcq.lisp

```diff
@@ -66,8 +66,14 @@ The input strings are assumed to be FORMAT-compatible, so sequences like ~<newli
 (defmethod %serialize (payload)
   payload)
 
+(defmethod %serialize ((payload array))
```

Comment:
this should be `vector` not `array`, since the `map` operation doens't make sense otherwise